csclient/params/params.go

```diff
@@ -173,6 +173,23 @@ type SearchResponse struct {
 	Results    []SearchResult
 }
 
+// ListResult holds a single result from a list operation.
+type ListResult struct {
+	Id *charm.URL
+	// Meta holds at most one entry for each meta value
+	// specified in the include flags, holding the
+	// data that would be returned by reading /meta/meta?id=id.
+	// Metadata not relevant to a particular result will not
+	// be included.
+	Meta map[string]interface{} `json:",omitempty"`
+}
+
+// ListResponse holds the response from a list operation.
+type ListResponse struct {
+	Total      int
```

Comment:
In search this shows the total number of matches even if they aren't all listed, what does it do here?